database/migrations/2023_10_31_204320_create_wikibase_notification_sent_records_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWikibaseNotificationSentRecordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wikibase_notification_sent_records', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+
+            $table->string('notification_type') -> nullable();
```

Comment:
This table still needs a `wiki_id` foreign key so we can map it to a wiki.

See here for an example https://github.com/wbstack/api/blob/5a8bc9f2ac07d223cb5124d434f400d061b1246b/database/migrations/2023_09_19_094751_create_wiki_lifecycle_events_table.php#L22-L23